config/webpack.common.js

```diff
@@ -11,7 +11,8 @@ export default {
 
   entry: {
     app: './_src/index.js',
-    admin: './_src/admin.js'
+    admin: './_src/admin.js',
+    helpcenter: './_src/helpcenter.scss'
```

Comment:
Doesn't webpack generate the CSS with a timestamp / hash to avoid overwriting existing files? Or is it not set up that way for this site?

Reply:
Good question! I figured this would work because https://www.controlshiftlabs.com/assets/app.css seems to work. In the preview it seems like it comes out right: https://deploy-preview-196--cranky-wiles-d63140.netlify.app/assets/helpcenter.css